### PR TITLE
fix: correctly set isolation flag

### PR DESCRIPTION
**Pull request by sakulstra**

### CODE REVIEW

File: packages/math-utils/src/formatters/user/index.ts [hunk 1]
```diff
@@ -128,14 +129,14 @@ export function formatUserSummary({
...
-  const isInIsolationMode = Boolean(
-    rawUserReserves.find(
-      reserve =>
-        reserve.reserve.debtCeiling !== '0' &&
-        reserve.scaledATokenBalance !== '0',
-    ),
+  const isolatedUserReserve = formattedUserReserves.find(
+    reserve =>
+      reserve.reserve.debtCeiling !== '0' &&
+      reserve.usageAsCollateralEnabledOnUser,
```
Comment (sakulstra) [inline packages/math-utils/src/formatters/user/index.ts:135]: it's not the balance, but collateralEnabled that's relevant

File: packages/contract-helpers/src/index.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export {
   PERMISSION_MAP,
 } from './permissions-manager/types/PermissionManagerTypes';
 export * from './v3-UiIncentiveDataProvider-contract';
+// export * from './ui-incentive-data-provider';
```
Comment (defispartan) [inline packages/contract-helpers/src/index.ts:7]: Why is this removed? We still need this for V2 incentives

Comment (sakulstra) [reply packages/contract-helpers/src/index.ts:7]: seems like we don't need it as here is just added a comment for some historical reason. will remove the comment though
